src/ws/session.c

```diff
@@ -533,6 +537,56 @@ pam_conv_func_dummy (int num_msg,
   return PAM_CONV_ERR;
 }
 
+static void
+create_s4u_ticket (const char *username)
+{
+  int res;
+  static struct passwd pwd_buf;
+  static char pwd_string_buf[8192];
+  struct passwd *pwd;
+  OM_uint32 major, minor;
+  gss_buffer_desc user_buf = { .length = strlen (username), .value = (char *) username };
+  gss_cred_id_t server_cred = GSS_C_NO_CREDENTIAL;
+  gss_name_t impersonee = GSS_C_NO_NAME;
+
+  res = getpwnam_r (username, &pwd_buf, pwd_string_buf, sizeof pwd_string_buf, &pwd);
+  if (pwd == NULL)
+    {
+      warnx ("couldn't load user info for: %s: %s", username, res == 0 ? "not found" : strerror (res));
+      goto out;
+    }
```

Comment:
Why?  If we're convinced already that we want to issue a ticket for this username then surely we already know that it exists... It's not like you use any of the return values for anything...

Reply:
Oops, yes -- that's a leftover from an ealier approach, will remove.

Reply:
Dropped.